src/app/firebase-configs.js

```diff
@@ -31,4 +31,8 @@ export const googleAuthProvider = new firebase.auth.GoogleAuthProvider();
 export const signInWithEmailAndPassword = (...args) =>
   firebaseApp.auth().signInWithEmailAndPassword(...args);
 
+export const sendEmailVerification = () => {
+  firebaseApp.auth().currentUser.sendEmailVerification();
```

Comment:
use an inline arrow (like `signInWithEmailAndPassword`), so this proxy function becomes async like its prime function.